### DISCUSSION TITLE
GP12 1080 quality vs fps

### DISCUSSION THREAD

**Discussion by Madsfoto** [gopro/labs#1193] [Q&A] [answered]
I am unsure if this is the correct place, to ask about the encoding of video

The issue I have is that in 1080p 24fps, 1080p 30fps, 1080p 120fps, 1080p 240fps and 2.7k 240fps I get a distinct softening of the recording, but in 1080p 60 fps (and all 2160p and 2988) the image is sharp. 

I have installed labs 2.30.70 and added BITR=180 and NR01=01.
Screenshots attached
![Screenshot 2024-12-03 150917](https://github.com/user-attachments/assets/270c4232-ced9-4adc-80a8-048df28632cc)
![Screenshot 2024-12-03 150921](https://github.com/user-attachments/assets/b3a82d0e-958b-4aa9-805d-d609dc899a68)

**Comment by dnewman-gpsw** [comment, marked as answer]
Not really a Labs question. Likely sensor binning is used on some 1080 modes. These are my guesses, as I'm not on that dev/product team. Seems potentially unnecessary for 1080p24/30 that could have been like 1080p60, but very much makes sense at 120 and 240fps. This is a power saving requirement, and for 240fps a technical must. With sensor binning, the camera's internal image processing would around 2.7k, versus 5.3k.  Processing 5.3k for a 2K (1080) image is mostly overkill, but there are some advantages to oversampling. For this reason, it is better on any camera to shot at the highest resolution for the frame rate you want, and down-sample in post (a GPU can do it better than any camera.) 1080p24 is the most power efficient on HERO12, but 4K24 or 5.3K24 looks better when down-sampled. On HERO13 there is no 1080p24, as 24p users generally favor quality battery over runtime (4K24 is the minimum for 24fps.) So for 1080 below 50/60, the assumption must be that users want longer run time, so sensor binning is used.
